### PR TITLE
added footer component

### LINKED ISSUE #6: added footer
added footer to home page with some social icons

### PR DESCRIPTION
**Pull request by mariyasada**
added footer component to the landing page
footer
Use <a> tag instead of a Link because Link doesn't redirect to social icon's website

### CODE REVIEW

**Review by Bhakti199 — APPROVED** [review]
- Code is nicely written and structured properly.
- Proper use of naming coventions
- I like that you break down code into small components properly.
 This PR is good to merge

File: src/components/Footer/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  return (
+   <div className="footer">
+      <div className="footer-heading">Made by Mariya Sada | © | 2022</div>
+      <ul className="nav-items">
+        <li className="nav-item">
+          <a href="https://github.com/mariyasada">
+            <span className="nav-icon">              
+             <BsGithub title="Github" />
+            </span>
+          </a>
```
Comment (JantuDeb) [inline src/components/Footer/Footer.jsx:15]: You have imported the link component but have not used it.

**Review by JantuDeb — APPROVED** [review]
Added a comment

File: src/components/Footer/Footer.jsx (hunk 1 above)
Comment (mariyasada) [reply src/components/Footer/Footer.jsx:15]: yes when i use Link instead of a it doesn't redirect to social website that's why i a\have not used

**Pull request merged**